src/SFA.DAS.Commitments.Database/Tables/ApprenticeshipFlexibleEmployment.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].ApprenticeshipFlexibleEmployment
+(
+	[ApprenticeshipId] BIGINT NOT NULL,
+	[EmploymentPrice] INT NOT NULL,
+	[EmploymentEndDate] DATETIME2 NOT NULL,
+    CONSTRAINT [PK_FlexibleEmployment] PRIMARY KEY ([ApprenticeshipId])
+)
+GO
```

Comment:
My fault here, but I don't think we can enforce the EmploymentPrice and Employment EndDate to be NOT NULL here. The user can possibly leave one or both of these values as empty. Therefore both fields must be NULLABLE

Reply:
Yes, you're right.  I'd made the class model nullable but forgot the db.